MANIFEST/edge_detection.manifest.yaml

```diff
@@ -0,0 +1,9 @@
+COMMAND:
+  - name: Edge Detection
+    key: EDGE_DETECTION
+    type: IMAGE_PROCESSING
+    pip_dependencies:
+      - name: opencv-python-headless
+        v: 4.7.0.72
```

Comment:
Could you please wrap them in quotation marks so that zod doesn't complain?

Reply:
The package names?